graphix/sim/statevec.py

```diff
@@ -495,6 +503,16 @@ def expectation_value(self, op, qargs):
         st1.evolve(op, qargs)
         return np.dot(st2.psi.flatten().conjugate(), st1.psi.flatten())
 
+    def subs(self, variable, substitute) -> Statevec:
+        """Return a copy of the state vector where all occurrences of
+        the given variable in measurement angles are substituted by
+        the given value.
+
+        """
```

Comment:
is it correct to understand this can be done only if `sympy`-version of parameter (e.g. graphix-parameter module) is used? if so, could you mention that in the docstring?

Reply:
Fixed in af74a37 .